README.md

```diff
@@ -13,9 +13,9 @@ applications' configurations, leveraging easy application integration.
 
 ## Often used links ##
 
-- [build server](http://build.libelektra.org:8080/)
-- [tutorials](http://git.libelektra.org/blob/master/doc/tutorials/)
-- [API docu](http://doc.libelektra.org/api/latest/html/)
+- [build server](https://build.libelektra.org/)
```

Comment:
Looks like it. I've never used merge for this branch, and tried rebasing by using "git rebase ElektraInitiative/master" while i'm in this branch, but that somehow lead to this merge commit b970a1f ... isn't this the way to rebase things? It hasn't told me about any conflicts and hasn't asked me to resolve anything either so i wonder where this commit comes from.